### PR TITLE
WIP: blog-post for week 30

### PR DESCRIPTION
**Pull request by nikromen**

### CODE REVIEW

File: content/posts/July-2021.md [hunk 1]
```diff
@@ -37,3 +37,12 @@ is either internal (not user facing) or not finished yet.
   Don't forget to give our GitHub application permissions to use it.
   You can expect more improvements in this field.
   ([packit-service#1167](https://github.com/packit/packit-service/pull/1167))
+
+## Week 30 (July 26th - July 30th)
+
+- Source-git patch metadata: the `squash_commits` key is deprecated.
```
Comment (lbarcziova) [inline content/posts/July-2021.md:43]: if we want to mention this, we should also describe the way it is done currently, you can check Hunor's [doc changes](https://github.com/packit/packit.dev/pull/316/files) about this

File: content/posts/July-2021.md [hunk 2]
```diff
@@ -37,3 +37,12 @@ is either internal (not user facing) or not finished yet.
...
+  ([packit#1309](https://github.com/packit/packit/pull/1309))
+- `packit init` can now find a spec file inside of a git repository and set
+  it in `packit.yaml` ([packit#1313](https://github.com/packit/packit/pull/1313))
+- Status labels on dashboard have been reworked.
```
[the remark this answers is not in the document]
Comment (mfocko) [reply content/posts/July-2021.md:47]: You can „steal“ it from the PR, at the end of description are release notes. Adjust if you need to :)

link: packit/dashboard#128

**Comment by nikromen** [comment]
recheck

**Review by TomasTomecek — APPROVED** [review]

**Review by lbarcziova — APPROVED** [review]
thanks!

**Pull request merged**